cosmic-settings/src/pages/desktop/mod.rs

```diff
@@ -31,9 +37,21 @@ impl Default for Page {
             },
         );
 
+        let comp_config = cosmic_config::Config::new("com.system76.CosmicComp", 1).unwrap();
+        let show_active_hint = comp_config
+            .get("active_hint")
+            .inspect_err(|err| {
+                if !matches!(err, cosmic_config::Error::NoConfigDirectory) {
+                    error!(?err, "Failed to read config 'active_hint'")
+                }
+            })
+            .unwrap_or(true);
```

Comment:
I'm unsure here if there's a better solution than retrieving the compositor config extra for this page.
I'm open for suggestions!